test/scenarios/el94-src@upgrade-fails-on-1st-boot-but-recovers-on-2nd.sh

```diff
@@ -4,7 +4,7 @@
 
 scenario_create_vms() {
     prepare_kickstart host1 kickstart.ks.template rhel-9.4-microshift-source
-    launch_vm host1
+    launch_vm host1 rhel-9.4
```

Comment:
We cannot use `rhel-9.4` because it will use rhel-9.4.iso where we have problem with first boot not having persistent journal thus breaking the test (which wants to check amount of boots, but the first one is snapped from the reality).
Same for `el94-src@upgrade-fails`